WalletWasabi.Fluent/ViewModels/AddWalletPageViewModel.cs

```diff
@@ -35,6 +35,12 @@ public class AddWalletPageViewModel : NavBarItemViewModel
 				NavigateTo(new RecoverWalletViewModel(navigationState, WalletName, network, walletManager), NavigationTarget.DialogScreen);
 			});
 
+			ConnectHardwareWalletCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				await navigationState.DialogScreen.Invoke().Router.Navigate.Execute(
+					new ConnectHardwareWalletViewModel(navigationState, network, walletManager));
```

Comment:
`navigationState.DialogScreen.Invoke().Router.Navigate.Execute(
					new ConnectHardwareWalletViewModel(navigationState, network, walletManager));`
to
`NavigateTo(new ConnectHardwareWalletViewModel(navigationState, network, walletManager), NavigationTarget.DialogScreen);`

Reply:
Done